src/utils.js

```diff
@@ -69,3 +69,64 @@ export const EventUtil = {
 		return recontructedEvents;
 	},
 };
+
+export const NewsUtil = {
+	fetchBookingNews: () => {
+		const url =
+			"/ocs/v2.php/apps/activity/api/v2/activity/calendar?format=json";
+		return axios
+			.get(url)
+			.then((resp) => {
+				if (resp.status !== 200) throw new Error("Error fetching news");
+
+				const bookingNews = resp.data.ocs.data.filter(function (elm) {
+					return (
+						elm.type === "calendar_event" &&
+						elm.subject.includes("You updated event ✔️")
+					);
+				});
```

Comment:
elm => elm.type === 'calendar_events' && elm.subject.includes("You updated event ✔️")